test/Models/ArcGisFeatureServerCatalogItemSpec.ts

```diff
@@ -120,4 +136,189 @@ describe("ArcGisFeatureServerCatalogItem", function() {
       }
     });
   });
+
+  describe("esriSLS", function() {
+    it("properly loads features", async function() {
+      runInAction(() => {
+        item.setTrait(
+          CommonStrata.definition,
+          "url",
+          featureServerUrlStyleLines
+        );
+      });
+      await item.loadMapItems();
+
+      expect(item.geoJsonItem).toBeDefined();
+      if (isDefined(item.geoJsonItem)) {
+        const geoJsonData = item.geoJsonItem.geoJsonData;
+        expect(geoJsonData).toBeDefined();
+        if (isDefined(geoJsonData)) {
+          expect(geoJsonData.type).toEqual("FeatureCollection");
+          const features = <JsonArray>geoJsonData.features;
+          expect(features.length).toEqual(13);
+        }
+      }
+    });
+
+    it("properly styles features", async function() {
```

Comment:
It would be nice if you can add some color style tests similar to https://github.com/TerriaJS/terriajs/pull/4412.
The old code has a bug where a polygon outline color is set to be the same as filled color, making it invisible from a particular viewing angle. (See issue https://github.com/TerriaJS/terriajs/issues/4420.) Adding the tests will prevent similar errror from happening in the future.

Reply:
I can't commit to #4412 branch so that is the why I am committing to my branch, otherwise, I would move already to the one in #4412